tests/kafkatest/services/kafka/kafka.py

```diff
@@ -628,7 +635,7 @@ def start_minikdc_if_necessary(self, add_principals=""):
     def alive(self, node):
         return len(self.pids(node)) > 0
 
-    def start(self, add_principals="", nodes_to_skip=[], timeout_sec=60, **kwargs):
+    def start(self, add_principals="", nodes_to_skip=[], isolated_controllers_to_skip=[], timeout_sec=60, **kwargs):
```

Comment:
can we just put the controllers we want to skip into `nodes_to_skip`?

Reply:
hm, feel free to correct me if I'm wrong - it doesn't seem like it will work because we specifically rely on `self.kafka.start()` for our test (which will also create and start the isolated controller quorum as part of the startup process) and `self.kafka.start()` already is using `nodes_to_skip` to understand which broker-only nodes it needs to skip startup for in this scenario

Reply:
it seems a bit messy to change the meaning of `nodes_to_skip` to mean "skip isolated controller nodes instead of broker nodes if in isolated mode", so I decided to just add another parameter to make it very clear

Reply:
ok.